src/test/lib/expression/evaluation/like_matcher_test.cpp

```diff
@@ -54,4 +55,19 @@ TEST_F(LikeMatcherTest, NotMatching) {
   EXPECT_FALSE(match("Hello", "He_o"));
 }
 
+TEST_F(LikeMatcherTest, LowerUpperBound) {
+  const auto pattern = pmr_string("Japan%");
+  const auto [lower_bound, upper_bound] = LikeMatcher::get_lower_upper_bound(pattern);
+  ASSERT_EQ(lower_bound, "Japan");
+  ASSERT_EQ(upper_bound, "Japao");
+
+  // Check that if the char ASCII value before the wildcard has the max ASCII value 127, the upper and lower bound are
+  // the same.
+  auto max_ascii_value = pmr_string(1, static_cast<char>(127));
```

Comment:
Aren't those two separate test cases? Also, should we include leading wildcards, see above?

Reply:
I separated the test case and added a new one for leading wildcards.